### ISSUE TITLE
question about the returned errors 

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#10231]
1. 96097490-88cd-11ef-bca3-97afaa522f81. These orders all call the **shipment create** and return errors `Carrier\u0027s service could not handle transaction. Try again`，"commandId":"19918992-82cc-473b-b35b-81711a1b4698"
2. 9d486950-9b42-11ef-a16e-7d69ff3d08b6. It returns `”Entered text exceeds 35 characters“`,"commandId":"ee17b8c0-a3b6-4208-b833-41e95164e423". But where the text exceeds 35 characters？

**Comment by AureliuszBrussy** [comment]
1. This command is pretty old right now. Unfortunately, we no longer have logs available that shows the reason for the failure.
2. We have returned the path in which you exceeded the length. Error response: `path=receiver.street`. The problem is that [we are currently preparing to remove](https://developer.allegro.pl/news/ship-with-allegro-%20on-january-31-2025-we-will-remove-the-streetNumber-field-6M7kKbzw5CB) the `streetNumber` field. Currently, in the transition stage, the content from `streetNumber` is moved to `street`. For the moment, it is best if you stop sending any value in the `streetNumber` field. Then this error will not occur.

**Comment by yangcen6** [comment]
2. So now there is no need to pass the street number？

**Comment by AureliuszBrussy** [comment]
Yes, now you can pass the complete address in `street` field. You do not need to parse it and separate street and street number.

**Comment by yangcen6** [comment]
So this means that street number is not a required field?

**Comment by AureliuszBrussy** [comment]
Yes, it is not required.

**Comment by yangcen6** [comment]
For Q1 ,you can see the "commandId":"0327a530-5547-4b04-bb93-a2c1f870d9e8"

**Comment by AureliuszBrussy** [comment]
It's the same case as with "19918992-82cc-473b-b35b-81711a1b4698"—unfortunately, we no longer have logs available that show the reason for the failure.
